central/complianceoperator/v2/report/manager/reportgenerator/consts.go

```diff
@@ -0,0 +1,13 @@
+package complianceReportgenerator
+
+const (
+	defaultEmailBodyTemplate = "{{.BrandedPrefix}} has identified non compliant profile checks for clusters scanned by your \n" +
```

Comment:
I think the `non compliant` part needs to be templated.  While 99.9999999% of the time the results will be non compliant there is always a chance that everything is in compliance.